packages/tasks/src/model-libraries.ts

```diff
@@ -831,6 +831,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"model_vae_fp16.pt"`,
 		snippets: snippets.threedtopia_xl,
 	},
+	anemoi: {
```

Comment:
Can you move this snippet to the top of the mapping to respect alphabetical order?

Reply:
Sure, does the library and the snippet need to be moved?